ap-plus/config/_colour.scss

```diff
@@ -0,0 +1,16 @@
+// Black
+$black: #000;
+
+// White
+$white: #fff;
+
+// Red
+$red: #b8202e; // APW $red (used for error styles)
```

Comment:
Not raving about using a colour they've not given us here – can we use the maroon instead?